pkg/acsengine/engine.go

```diff
@@ -283,10 +291,26 @@ func prepareTemplateFiles(properties *api.Properties) ([]string, string, error)
 	return files, baseFile, nil
 }
 
-func getParameters(properties *api.Properties, isClassicMode bool) (map[string]interface{}, error) {
+//GetCloudSpecConfig returns the kubenernetes container images url configurations based on the deploy target environment
+//for example: if the target is the public azure, then the default container image url should be gcr.io/google_container/...
+//if the target is azure china, then the default container image should be mirror.azure.cn:5000/google_container/...
+func GetCloudSpecConfig(location string) AzureEnvironmentSpecConfig {
+	cloudSpecConfig := AzureCloudSpec
+	switch location {
```

Comment:
For now, let's assume that they can use the public mirrors. I'd still prefer that we either include the other clouds in the switch, or remove them from being explicit clouds. See what I mean? Otherwise someone is going to come along, change the AzureGermanCloud definition, and they may think they're done. But because AzureGermanCloud isn't part of the switch... they'll never actually get the updated values. So we should either go ahead and add it to the switch, or remove those environment definitions so that if  someone comes along and adds it later, they're more likely to look for the switch statement and add the new cloud.